crypto/asn1/x_long.c

```diff
@@ -12,6 +12,9 @@
 #include <openssl/asn1t.h>
 #include <openssl/bn.h>
 
+#if !(OPENSSL_API_COMPAT < 0x10200000L)
```

Comment:
There places where we use >=. I did it this way so the expression is kept similar to the one in asn1t.h...